src/components/UI/Pagination/Pagination.tsx

```diff
@@ -97,7 +97,7 @@ export const Pagination: FunctionComponent<PaginationProps> = ({ totalNoOfPages,
           onClick={goNextPage}
           disable={currentPage >= totalNoOfPages}
           data-testid="page-next"
-          className="hover:bg-grey-100 hover:text-grey"
+          className="hover:bg-blue hover:text-white hover:rounded-md"
```

Comment:
do the same thing as the other error to change the background color accordingly to the disable state.

Reply:
done.

Reply:
ok